### PR TITLE
Add SplitFastq module

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
# nf-core/sarek pull request

Many thanks for contributing to nf-core/sarek!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/sarek branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/sarek)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -448,6 +451,19 @@ workflow {
 
     // STEP 1: MAPPING READS TO REFERENCE GENOME WITH BWA-MEM
 
+    //TODO: not sure whether to split the reads before qc or after
```
Comment (MaxUlysse) [inline main.nf:454]: I think so far we were splitting before QC.
Not sure what would be the best option...

Comment (FriederikeHanssen) [reply main.nf:454]: Not sure either. I will fix the merging first, that is where it fails currently and then think about wether it should be moved up or not.

Comment (MaxUlysse) [reply main.nf:454]: we can ask the community about it actually

Comment (FriederikeHanssen) [reply main.nf:454]: Hm, if we do it before the QC, QC would run on each pair individually. Are the reports merged?

Comment (MaxUlysse) [reply main.nf:454]: Not sure we can merge reports.
And it would make sense to have the QC on the fastqs we're starting with actually

**Pull request closed without merging**